homeassistant/components/xbox/base_sensor.py

```diff
@@ -49,13 +49,15 @@ def entity_picture(self) -> str:
         # Xbox sometimes returns a domain that uses a wrong certificate which creates issues
         # with loading the image.
         # The correct domain is images-eds-ssl which can just be replaced
-        # to point to the correct image, with the correct domain and certificate
-        # using YARL URL lib, we can replace the domain part of a url and append
-        # only the queries we need (url and format) and omit unnecessary ones (padding)
+        # to point to the correct image, with the correct domain and certificate.
+        # Using YARL URL lib, we can replace the domain part of an url and remove
+        # the padding query.
```

Comment:
Don't mention Yarl here, i can read what you are doing. What i don't know is why padding is removed.

Reply:
Honestly, i don't know why padding needs to be removed. With it, the image does not load because of a 400 error message, but the cause of it i don't know. maybe @hunterjm does understand? 

Another commit is coming because the query is wrong, the padding query is 'mode=Padding' so i need to pop mode, not padding.